src/main/java/net/greatstart/dto/DtoUserProfile.java

```diff
@@ -16,4 +16,5 @@
     private String email;
     private String lastName;
     private byte[] photo;
+    private String initial;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "initial" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)